performance/whynot.benchmark.ts

```diff
@@ -0,0 +1,25 @@
+import runner from './BenchmarkRunner';
+import * as whynot from '../src/index';
+
+let vm: whynot.VM<number, void, void>;
+let input: number[];
+runner.addBenchmark(
+	'simple',
+	() => {
+		vm.execute(input);
+	},
+	() => {
+		vm = whynot.compileVM(assembler => {
+			assembler.jump([1, 7]);
```

Comment:
Let's write down what this does. Usually when we write Whynot programs, we push into the data of jumps to allow us to add some variable names, this makes the programs more understandable

Reply:
I agree but don't think it's best if I go through the effort. I only moved this test from one place to the other and also find it very hard to understand what the flow of this test really is.
@bwrrp if you can explain it than I can comment the steps in this PR.

Reply:
I'll add a commit with a few comments